doc/09-object-types.md

```diff
@@ -34,6 +34,7 @@ the [Icinga 2 API](12-icinga2-api.md#icinga2-api-config-objects).
   templates                 | Array                 | Templates imported on object compilation.
   package                   | String                | [Configuration package name](12-icinga2-api.md#icinga2-api-config-management) this object belongs to. Local configuration is set to `_etc`, runtime created objects use `_api`.
   source\_location          | Dictionary            | Location information where the configuration files are stored.
+  name                      | String                | Object name. Might be used for advanced usage in [apply rules](03-monitoring-basics.md#using-apply).
```

Comment:
"used for [...] usage" sounds odd to me. Also, if we're already rephrasing here: is `assign where match("...", host.name)` really that advanced (it links to monitoring basics after all)? So a simple "Might be used in apply rules" would do for me.

Reply:
Ack. The sentence got a bit messed up during refactoring.

I have changed this and used your suggestion.